experimental/traffic-portal/src/app/core/servers/update-status/update-status.component.ts

```diff
@@ -106,31 +94,29 @@ export class UpdateStatusComponent implements OnInit {
 	 *
 	 * @param e The submission event.
```

Comment:
This param no longer exists (but re: my earlier comment about forms,  it should)